package.json

```diff
@@ -28,6 +28,8 @@
   },
   "homepage": "https://github.com/GSG-G8/courses-search#readme",
   "dependencies": {
+    "axios": "^0.19.2",
+    "cheerio": "^1.0.0-rc.3",
```

Comment:
**these modules should be installed inside `/server/package.json `not here and `axios` has been already installed**